plugin/init.lua

```diff
@@ -88,6 +88,15 @@ local function execute_cmd_with_stdin(cmd, input)
 			return success, stderr
 		end
 	else
+		local stdout = io.popen(cmd .. " 2>&1", "r")
+		if not stdout then
+			return false, "Failed to execute: " .. cmd
+		end
+		local err = stdout:read("*all")
+		stdout:close()
+		if err ~= "" then
```

Comment:
The idea is that if the output from reading from stdout after the process has finished produces some text, then we know it is stderr output? A comment  explaining why there is a read after the process has finished, would be great here, as this can be hard to know what is going on here :smile: 
Have you tested this?

Reply:
Yes - the `local stdout = io.popen(cmd .. " 2>&1", "r")` redirects stderr to stdout. This bit of code is testing that the cmd will work before executing it. It was the only way I was able to return errors if there was an issue with the cmd. I tested it by using `rage` instead of `age` (I don't have `rage` in my PATH), and was able to get the expected error.

Reply:
I see, it makes sense. If you could add a comment about this, then it would help with maintaining :smile:

Reply:
Yes, I'll add comments. Windows doesn't support `/dev/stdin` so I won't be able to do this check on Windows unfortunately. I'll add some content to the `README` for Windows users about the lack of error messaging.